README.md

```diff
@@ -107,6 +107,37 @@ end
 run(app)
 ```
 
+### Hanami
+
+Place your configuration under `config/initializers/kiev.rb`:
+
+```ruby
+require "kiev"
+
+Kiev.configure do |config|
+  config.app = :my_app
+  config.development_mode = Hanami.env?(:development)
+  config.log_path = File.join("log", "structured.log")
+end
+```
+
+Within your `MyApp::Application` file, include the `Kiev::Hanami` module, in order to register the middleware stack:
+
+```ruby
+module MyApp
+  class Application < Hanami::Application
+    include Kiev::Hanami
+
+    configure do
+      ##
```

Comment:
save the :evergreen_tree: by turning this 4-line comment into a one-line comment (and maybe explain that the `include` precedes the `configure` for a purpose, otherwise there's no need for this block at all in my opinion).